docs/platforms/apple/guides/ios/session-replay/index.mdx

```diff
@@ -6,6 +6,12 @@ notSupported:
 description: "Learn how to enable Session Replay in your mobile app."
 ---
 
+<Alert level="warning">
+
+Before enabling Session Replay in production, verify your masking configuration to ensure no sensitive data is captured. Our default settings aggressively mask potentially sensitive data, but if you modify these settings, you must thoroughly test your application. If you find any masking issues or sensitive data that should be masked but isn't, please [create a GitHub issue](https://github.com/getsentry/sentry-cocoa/issues/new/choose) and avoid deploying to production with Session Replay enabled until the issue is resolved.
```

Comment:
"but if you modify these settings, you must thoroughly test your application"

I would argue, not only then, IMO it's generally advisable, especially after they bump e.g. UI framework or system SDK versions